tools/show_ox_propolis.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+if [[ -n $1 ]]; then
+    UUID=$1
+else
+    # Use this hard coded UUID, which is probably not what you want.
+    UUID=2effac35-a8ba-45ae-966d-4487e0df0ad6
```

Comment:
should be `d7a01b55-93a6-c873-b341-d03c93385e97`

Reply:
Removing the default UUID, so we don't have to argue.